connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -185,10 +192,9 @@ public void close() {
         deleteAllTopics(backup.kafka());
         primary.stop();
         backup.stop();
+        assertFalse(exited.get());
```

Comment:
```suggestion
        assertFalse(exited.get());
        Exit.resetExitProcedure();
```

Reply:
Ah, fancy GitHub stuff. I pushed up the fix independently.